NGUIButtonInsetsExample/ViewController.swift

```diff
@@ -274,44 +289,104 @@ final class ButtonView: UIView {
     }
 }
 
+// MARK:  -
+// MARK: VC -
+
 class ViewController: UIViewController {
-    private let stackView = UIStackView()
-    private let buttonView = ButtonView()
-    private let resetButton = UIButton()
+    
+    // MARK: UI Properties -
+    
+    private lazy var buttonsView: ButtonsView = {
+        let bsv = ButtonsView()
+        bsv.translatesAutoresizingMaskIntoConstraints = false
+        bsv.accessibilityLabel = "ButtonsView"
+        return bsv
+    }()
+    
+    private lazy var resetButton: UIButton = {
+        let rsb = UIButton()
+        rsb.translatesAutoresizingMaskIntoConstraints = false
+        rsb.setTitle("Reset", for: .normal)
+        rsb.setTitleColor(.blue, for: .normal)
+        rsb.addTarget(self, action: #selector(resetTapped), for: .touchUpInside)
+        return rsb
+    }()
+    
+    private lazy var stackView: UIStackView = {
+        let stv = UIStackView()
+        stv.axis = .vertical
+        stv.spacing = 50
+        stv.translatesAutoresizingMaskIntoConstraints = false
+        stv.accessibilityLabel = "Main StackView"
+        stv.backgroundColor = .lightGray
+        return stv
+    }()
+    
+    private lazy var allInsetsView: AllInsetsView = {
+        let aiv = AllInsetsView(insetsDidChange: { [weak self] type, insets in
+            guard let `self` = self else { return }
+            self.buttonsView.set(insets: insets, type: type)
+        })
+        aiv.translatesAutoresizingMaskIntoConstraints = false
+        aiv.accessibilityLabel = "AllInsetsView"
+        aiv.backgroundColor = .lightGray
+        return aiv
+    }()
+    
+    //MARK: Properties -
+    
     private var timer: Timer?
+    
+    // MARK: Lifecycle -
 
     override func viewDidLoad() {
         super.viewDidLoad()
-        view.backgroundColor = .white
+        setupView()
+    }
+}
 
-        view.addSubview(stackView)
-        stackView.axis = .vertical
-        stackView.spacing = 50
+// MARK: Setup -
 
-        stackView.translatesAutoresizingMaskIntoConstraints = false
+private extension ViewController {
+    
+    func setupView() {
+        view.backgroundColor = .white
+        setupStackview()
```

Comment:
Could we make the naming consistent? This would be `setupStackView`

Reply:
Sure